js/page-specific/citysourced-viewer-googleMaps.js

```diff
@@ -1,237 +1,153 @@
-
-		/**
-		 * Creates the map, and renders it in the mapElementId element.
-		 */
-		createMap = function (mapElementId, settings) {
-			window.map = new google.maps.Map(document.getElementById(mapElementId), settings);
-		},
-
-		/**
-		 * Creates the autocomplete, and attaches it to the autocompleteElementId element.
-		 */
-		createAutoComplete = function (autocompleteElementId, settings) {
-			autocomplete = new google.maps.places.Autocomplete(document.getElementById(autocompleteElementId), settings);
-		},
-
-		/**
-		 * Adds a marker to the map.
-		 */
-		createMarker = function (settings) { 
-			var markerSettings = {
-					map: window.map,
-					position: {
-						lat: settings.Latitude * 1,
-						lng: settings.Longitude * 1
-					},
-					animation: google.maps.Animation.DROP
-				};
-
-			if (settings.ImageUrl) 
-				markerSettings.icon = settings.ImageUrl;
-			
-			if (settings.ZIndex) {
-				markerSettings.zIndex = settings.ZIndex;
-				markerSettings.optimized = false;
-			}
-			
-			marker = new google.maps.Marker(markerSettings);
-
-			if(settings.InfoWindowHtml)
-				marker.html = settings.InfoWindowHtml;
-			
-			return marker;
-		},
-
-		/**
-		 * Clears a marker from the map.
-		 */
-		clearMarker = function () {
-			if (marker)
-				marker.setMap(null);
-		},
-
-		/**
-		 * Puts a market on the map.
-		 */
-		placeMarker = function(latitude, longitude, shouldUpdateTextbox, callback) {
-			clearMarker();			
-		},	
-
-		/**
-		 * Centers the map on the given latitude and longitude, them places a marker.
-		 */
-		centerMapOnLatLng = function(latitude, longitude) {
-			placeMarker(latitude, longitude, false, function(isCounty) {
-				if (isCounty) {
-					var center = new google.maps.LatLng(latitude, longitude);
-
-					window.map.panTo(center);
-					window.map.setZoom(14);
-
-					google.maps.event.trigger(window.map, 'resize');
-					map.setCenter(center);
-				}
-			});
-		},		
-
-		/**
-		 * Looks up the street address from the latitude and longitude.
-		 */
-		reverseGeocode = function (latitude, longitude, successCallback) {
-			require([
-				"esri/tasks/Locator",
-				'esri/geometry/Point'				
-			], function(Locator, Point) { 
-
-				var point = new Point(longitude, latitude);
-
-				var locatorSettings = {
-					countryCode: 'US',
-					outSpatialReference: spatialReferenceId,
-					url: geocodeServerUrlBCGIS
-				};
-
-				var locator = new Locator(locatorSettings);
-
-				var requestOptions = {
-					responseType: 'json'
-				};
-
-				locator.locationToAddress(point).then(successCallback, function(err) {
-					console.log(err);
-				});
-			});
+baltimoreCounty.pageSpecific.viewerGoogleMaps = (function viewerGoogleMaps() {
+	var marker;
+	var spatialReferenceId = 4269;
+	var geocodeServerUrlBCGIS = '//bcgis.baltimorecountymd.gov/arcgis/rest/services/Geocoders/AddressPoint_NAD83/GeocodeServer';
```

Comment:
I was only thinking of the Angular stuff when I made the constants, but you have a point. I can make a baltimoreCounty.pageSpecific.baltCoGo.constants() object and expose it through the angular module since it'll already be loaded.